6_predict/src/plot_diagnostics.R

```diff
@@ -447,6 +447,101 @@ make_class_prediction_map <- function(class_probs, reaches, out_dir,
   return(fnames)
 }
 
+
+make_transition_region_map <- function(class_probs, reaches,
+                                       region_threshold,
+                                       out_dir, model_name, pt_size = 0.5) {
+  #' @description this function creates a map showing the transition regions 
+  #' (reaches not distinctly predicted in single cluster)
+  #' 
+  #' @param class_probs dataframe of predicted class probabilities for each reach.
```

Comment:
This also expects that the columns are ordered 1, 2, ... 5 for the regions, right?

Reply:
@cstillwellusgs - checking if this is true